apps/sv/src/main/scala/org/lfdecentralizedtrust/splice/sv/store/SvDsoStore.scala

```diff
@@ -348,15 +348,19 @@ trait SvDsoStore
     ]]
   ]
 
-  final def getExpiredRewards(
+  /** Returns at most expired coupon batches per round and coupon type.
```

Comment:
```suggestion
  /** Returns at totalCouponsLimit most expired coupon batches per round and coupon type.
```
probably? otherwise I don't understand the sentence. The name sounds off though then as it's clearly not the total but the per round and coupon type max so maybe rename to batch size or something.

Reply:
also is there any reason why we split it by coupon type? That seems a bit pointless